.github/CODEOWNERS

```diff
@@ -1 +1,43 @@
-.github/workflows @SamMousa @brunobowden
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence. 
+#
+# Security sensitive items must be at the end.
+#
+# All user entries should be alpha-sorted and lowercase.
+
+# These owners will be the default owners for everything in
+# the repo, unless a later match takes precedence.
+# Sadly, we have to repeat the top-level owners everywhere,
+# since we are not team-members of a GitHub Organization.
+
+* @advaydev1 @brunobowden @crazybob @hspinks @sammousa
+
+# Feature areas:
+
+# Client UI
+/client/flutter/ @advaydev1 @brunobowden @crazybob @hspinks @patniemeyer @sammousa @theswerd
+
+# Client Localization
+# Pure translation changes can be approved by @sapte91, but Client UI folks can also still approve/submit
+# as needed when mixed with code changes.
+/client/flutter/lib/generated/intl/ @advaydev1 @brunobowden @crazybob @hspinks @patniemeyer @sammousa @sapte91 @theswerd
+/client/flutter/lib/l10n/ @advaydev1 @brunobowden @crazybob @hspinks @patniemeyer @sammousa @sapte91 @theswerd
+
+# Team-wide processes
+/docs/ @advaydev1 @brunobowden @crazybob @hspinks @deanhach
```

Comment:
It might be good to move `.github/issue_template.md` and `.github/pull_request_template.md` to the `/docs` directory so they are covered by this.

Reply:
/docs are about the product/team.  While .github to me is this GitHub-specific cruft.  But I don't have a strong opinion.

Top owners own the templates right now.

Reply:
Got it.

What do you mean by "top owners"? I don't see any rules for `/.github/` or `/`.

Reply:
* = top owners, i dunno the formal name